Graphs/Table/index.js

```diff
@@ -288,7 +288,7 @@ class Table extends AbstractGraph {
         });
 
         props.data.forEach( (d, i) => {
-            const random = this.generateRandom();
+            const random = hash(d);
```

Comment:
@nxanil This is still not going to work. Please use uniqueId here from lodash. The reason for this is that objectHash will still fail if a column has react element as value.